assets/dev/js/frontend/tools/stretch-element.js

```diff
@@ -55,11 +62,22 @@ module.exports = elementorModules.ViewModule.extend( {
 			correctOffset = -correctOffset;
 		}
 
+		// Consider margin
+		if ( settings.margin ) {
+			correctOffset += settings.margin;
+		}
+
 		var css = {};
 
-		css.width = containerWidth + 'px';
+		let width = containerWidth;
+
+		if ( settings.margin ) {
```

Comment:
Is this margin coming from a control. Could it not also be a format `2px 2px 2px 2px` or `2px`. or is it not from a margin (4 number input) control?

Reply:
It doesn't come from anywhere currently.
This is just groundwork for a future feature Aviv told me was planned, which was allowing adding a margin in Full width mode (mega menu). I used the opportunity that I'm working on this file to add this little infra feature, to save the need for a Core PR when that MM Pro feature comes.